overlord/snapstate/storehelpers.go

```diff
@@ -113,6 +113,11 @@ var installSize = func(st *state.State, snaps []minimalInstallInfo, userID int)
 		accountedSnaps[snap.InstanceName] = true
 	}
 
+	// if prerequisites are included in the install, don't download them again
```

Comment:
We aren't technically downloading them, only querying for size, the comment should probably be tweaked.